WooCommerce/Classes/Model/Coupon+Woo.swift

```diff
@@ -78,3 +78,30 @@ extension Coupon {
         }
     }
 }
+
+// MARK: - Sample Data
+extension Coupon {
+    static let sampleCoupon = Coupon(couponID: 720,
```

Comment:
nit: I wonder if it makes more sense to only have this sample data code in debug mode, or use Copiable like `Coupon.copy(...)` in its usage? Just thought it'd be nice only keeping production code in release builds.

Reply:
Great point! Since there's no easy way to initialize coupons without inputting values for each property, I'm keeping the code for the sample data in a check for debug mode in dbac04fa82ee94ea14221f8caea3f44de87a2c3d.